content/en/platform/corda/4.6/open-source/node-administration.md

```diff
@@ -163,7 +163,7 @@ To pass a security policy use `java -Dcapsule.jvm.args=-javaagent:./drivers/jolo
 
 When running in dev mode, Hibernate statistics are also available via the Jolkia interface. These are disabled otherwise
 due to expensive run-time costs. They can be turned on and off explicitly regardless of dev mode via the
-`exportHibernateJMXStatistics` flag on the [database configuration](corda-configuration-file.md#database-properties-ref).
+`exportHibernateJMXStatistics` flag on the [database configuration](corda-configuration-file.md).
```

Comment:
I think this should actually point here: https://tolkien.preview.docs.r3.com/en/platform/corda/4.6/open-source/corda-configuration-fields.html#database